docs/modules/language-reference/pages/index.adoc

```diff
@@ -3179,6 +3180,157 @@ This section discusses language features that are generally more relevant to tem
+When a property is declared in a module or class but is not provided an explicit default value, the property's default value becomes the type's default value.
+Similarly, when `Listing` and `Mapping` types are declared with explicit type arguments for their element or value, their `default` property amends that declared type.
+When `Listing` and `Mapping` types are declared without type arguments, their `default` property amends an empty `Dynamic` object.
+Some types, including `Pair` and primitives like `String`, `Number`, and `Boolean` have no default value; attempting to render such a property results in the error "Tried to read property `<name>` but its value is undefined".
+
+[source,{pkl}]
+----
+class Gizmo {
+  name: String = "doodad"
+}
+
+gizmo: Gizmo // <1>
+gizmoListing: Listing<Gizmo> // <2>
+gizmoMapping: Mapping<String, Gizmo> // <3> 
+----
+<1> Without an explicit default value, this property has default value `new Gizmo { name = "doodad" }`
+<2> With an explicit element type argument, this property's default value is equivalent to `new Listing<Gizmo> { default = new Gizmo { name = "doodad" } }`
+<3> With an explicit value type argument, this property's default value is equivalent to `new Mapping<String, Gizmo> { default = (_) -> new Gizmo { name = "doodad" } }`
+
+==== Explicit Parent
+
+Explicit amending is covered in detail by <<amending-objects, Amending Objects>>.
+Amending a property, entry, or value results in a new value with the same type as the parent.
+When amending a property that has not been previously declared, an empty `Dynamic` object is used as the parent.
+
+[source,{pkl}]
+----
+newProperty { // <1>
+  hello = "world"
+}
+amendedProperty = (newProperty) { // <2> 
+  hello = "there"
+}
+someMapping = new Mapping {
+  ["greetings"] { // <3>
+    hello = "everyone"
+  }
+}
+----
+<1> Amending a property.
+<2> Amending a value.
+<3> Amending a new `Mapping` entry.
+
+==== Explicitly Typed `new`
+
+Instantiating an object with `new <type>` results in a value that amends the specified type's default value.
+Notably, creating a `Listing` element or assigning a `Mapping` value with an explicitly typed `new` ignores a collection's `default` value.
+
+[source,{pkl}]
+----
+newProperty = new Dynamic { // <1>
+  hello = "world"
+}
+someListing = new Listing {
+  default {
+    isDefault = true
+  }
+  new Dynamic { // <2>
+    hello = "there"
+  }
+}
+someMapping = new Mapping {
+  default {
+    isDefault = true
+  }
+  ["greetings"] = new Dynamic { // <3>
+    hello = "everyone"
+  }
+}
+----
+<1> Assigning a `new` explicitly-typed value to a property.
+<2> Adding an `new` explicitly-typed `Listing` element.
+The value will not have property `isDefault = true` as the `default` property of the `Listing` is not used.
+<3> Assigning a `new` explicitly-typed value to a `Mapping` entry.
+The value will not have property `isDefault = true` as the `default` property of the `Mapping` is not used.
+
+==== Untyped `new`
```

Comment:
Even though untyped is technically correct, it may give the idea that the resulting value has no type. I think "Implicitly typed `new`", or perhaps, "Raw `new`" is a better name here.